requirements/edx/base.in

```diff
@@ -112,7 +112,7 @@ openedx-calc                        # Library supporting mathematical calculatio
 pdfminer.six                        # Used in shoppingcart for extracting/parsing pdf text
 piexif                              # Exif image metadata manipulation, used in the profile_images app
 Pillow                              # Image manipulation library; used for course assets, profile images, invoice PDFs, etc.
-py2neo<4.0.0                        # Used to communicate with Neo4j, which is used internally for modulestore inspection
+py2neo==4.0.0                       # Used to communicate with Neo4j, which is used internally for modulestore inspection
```

Comment:
@iam-mhaseeb 
Just to mention that edx handled this in a different way to avoid anyother issues
https://github.com/edx/edx-platform/blob/master/requirements/edx/base.txt#L14

edX is still using version `3.1.2` but they are getting it from `github`

Reply:
Thanks for pointing out @mumarkhan999. We are also going towards that and cherry-picking edx commit.